shared/actions/config/index.js

```diff
@@ -33,6 +34,79 @@ function getConfig (): AsyncAction {
   }
 }
 
+let followers = {}
```

Comment:
i was originally thinking it'd be nice to have a utility which you could call to just get this info w/o being in the context of a store. if its just reducers with access to this then we might not need it. we can remove this (or make it take in getState basically)